README.md

```diff
@@ -3,7 +3,7 @@
 [![node.js](https://img.shields.io/badge/node-%3E%3D%206.4.0-brightgreen.svg)](https://nodejs.org/en/)
 [![Gem](https://img.shields.io/gem/v/webpacker.svg)](https://github.com/rails/webpacker)
 
-Webpacker makes it easy to use the JavaScript preprocessor and bundler [Webpack](https://webpack.github.io)
+Webpacker makes it easy to use the JavaScript preprocessor and bundler [Webpack 2.x.x+](https://webpack.github.io)
```

Comment:
@dijonkitchen Lets just point all webpack links to  - https://webpack.js.org/ (we still have github.io one there)